charts/t8s-cluster/values.yaml

```diff
@@ -16,6 +16,9 @@ global:
       repository: bitnami/kubectl
       tag: 1.27.2
   injectedCertificateAuthorities: ""
+  kubeletExtraConfig:
+    # Only use this with k8s v1.27 and up
+    maxParallelImagePulls: 1
```

Comment:
Maybe we could set the default to be higher?

I can barely think of cases where you *wouldn't* want parallel image pulls

We could maybe do 4, 2 parallel pulls per core or something like that and then maybe just a flag to turn off the parallelization?